examples/sushi_dbt/config.py

```diff
@@ -1,10 +1,11 @@
 from pathlib import Path
 
-from sqlmesh.core.config import AirflowSchedulerConfig
+from sqlmesh.core.config import AirflowSchedulerConfig, DuckDBConnectionConfig
 from sqlmesh.dbt.loader import sqlmesh_config
 
 config = sqlmesh_config(Path(__file__).parent)
 
+trino_config = sqlmesh_config(Path(__file__).parent, state_connection=DuckDBConnectionConfig())
```

Comment:
did you mean to add this? If so, why is it the only extra config in here?

Reply:
This is a leftover from testing, but do you think it would be useful to have a config with a separate state connection so it's more convenient to run on trino/spark where we don't support main connection state? 

It's easy enough to add one if needed, though, so maybe just clutter?

Reply:
in general, we shouldn’t have testing code around.